presidio-analyzer/presidio_analyzer/predefined_recognizers/ahds_recognizer.py

```diff
@@ -0,0 +1,139 @@
+import os
+from typing import List, Optional
+
+from azure.health.deidentification import DeidentificationClient
```

Comment:
Please see how we do this with other optional recognizers, like Azure AI Language or Gliner. There is a try import step which would not fail if the package is not installed, but would throw and error if the user tries to instantiate the recognizer without first installing the packages.